G/GTK3/build_tarballs.jl

```diff
@@ -16,10 +16,7 @@ sources = [
 script = raw"""
 cd $WORKSPACE/srcdir/gtk+-*/
 
-# Temporary workaround #1 issue with apk
-apk add samurai
-
-# Temporary workaround #2: `util-linux` has a file, `/usr/bin/wall`, owned by
+# Temporary workaround: `util-linux` has a file, `/usr/bin/wall`, owned by
 # `root:tty`, but `apk` can't chown the file to the `tty` group because it
```

Comment:
Also the installation of `apk-tools` can be removed, and the `--no-chown` flag to `apk` isn't necessary anymore (it's automatically passed by the new [`apk` wrapper](https://github.com/JuliaPackaging/Yggdrasil/blob/ee4e1f70bd631b5316c4b6860f15995d0b16e5b7/0_RootFS/Rootfs/bundled/utils/apk_wrapper.sh))